.projenrc.js

```diff
@@ -5,7 +5,9 @@ const project = new typescript.TypeScriptProject({
   defaultReleaseBranch: 'main',
   deps: [
     'json2jsii',
+    'aws-cdk-lib',
```

Comment:
Can you remove these from `devDeps` so they aren't in two places?

Reply:
yes

Reply:
needed to revert that as those deps would not be present at build